README.md

```diff
@@ -9,13 +9,34 @@ Includes a Postman collection you can use to try out the [Sila sandbox API](http
 Make sure you have Python 3.6+. If using the quickstart, have bash, python venv, and the ability to run make commands from a Makefile.
 
 ## Quickstart (Mac and Linux)
+### Building the signer-server
+NOTE: the signer-server default port is **8181**
+#### Build and run from source
+1. Have a copy of **Python 3.6** installed; later versions of Python may be supported, but are untested.  We recommend [pyenv](https://github.com/pyenv/pyenv) for managing multiple Python versions. 
 1. Clone me into a local directory. Change directory into my root (same directory as `Makefile`).
-2. From the terminal, run `make venv`. This will install all dependencies into a local environment and only needs to be done once.
-3. From the terminal, run `make runserver`. By default, a local server will run on port 8181; you can specify a different port like `make runserver port=3001`.
-4. Use me with the REST client of your choice. `/forward` is the endpoint that signs requests and forwards them onward sans the original Authorization header. For convenience (not recommended for production use), you can generate new private key pairs with `/generate_private_key`, which also generates wallet verification signatures accepted by the Sila API when registering secondary wallets to users.
-5. If you want me to generate signature headers for you, set the Authorization header in requests to me like `Authorization: private-key; [signature header name]=[private key hex here]; [another signature header name]=[private key hex here]`. To have me forward your request somewhere, set a header like `X-Forward-To-URL: http://sandbox.silamoney.com/0.2/check_handle`.
-6. You can also set valid epoch timestamps and random UUID4 strings with the `X-Set-Epoch` and `X-Set-UUID` headers. You can specify which JSON key to set in the request like `X-Set-Epoch: header.created` or `X-Set-UUID: header.reference` (which each set keys in a nested dictionary like: `{"header": {"created": [sets epoch int in seconds]}}` and `{"header": {"reference": [sets random UUID4 string]}}`).
-7. Open issues or contact support if you need more help. For security vulnerabilities, please email support@silamoney.com rather than opening a GitHub issue.
+1. From the terminal, run `make venv`. This will install all dependencies into a local environment and only needs to be done once.
+1. From the terminal, run `make runserver`
```

Comment:
Probably change 1. 1. 1. to 1. 2. 3., etc.

Reply:
that would be on our end, whenever we make changes to the signerserver code; you'd update the `Version` label in the Dockerfile, then do another make.  Or, we could move the version out of the Dockerfile into any other file; it's just a `grep` in the Makefile
```
IMAGE_VERSION = $(shell grep "LABEL Version" Dockerfile | sed -e "s/.*=\"\(.*\)\"/\1/")
```

Reply:
Oh, I was talking about the numbering in the README, not versioning in the Dockerfile :D Sorry, should have made that more clear in my comment

Reply:
That's a markdown syntax, it's auto-numbering, will render them in ascending numerical order.  It's to support re-ordering, item insertion, etc.

![image](https://user-images.githubusercontent.com/29608541/91334477-dfd17680-e783-11ea-93cf-8a94337f59cd.png)

Reply:
ah, previewed the branch in github and that checks out!

Reply:
yeah, it's one of my favourite markdown features.  No MD preview in VsCode?

Reply:
ah, I was just not checking how it rendered like I should have - there's probably an extension for such a thing in VS Code, but I don't have it installed